cypress/e2e/signIn.cy.js

```diff
@@ -25,10 +24,16 @@ describe('Sign In page', () => {
     signInPage.typePassword(user.password);
     signInPage.clickSignInBtn();
 
-    homePage.assertHeaderContainUsername(user.username);
+    homePage.assertHeaderContainUsername(user.username); // Verify successful login
   });
 
   it('should not provide an ability to log in with wrong credentials', () => {
+    signInPage.visit();
+
+    signInPage.typeEmail('wrongemail@example.com');
+    signInPage.typePassword('WrongPassword123!');
+    signInPage.clickSignInBtn();
 
+    cy.contains('Invalid credentials').should('be.visible'); // Verify error for invalid login
```

Comment:
Verify that the application displays the 'Invalid credentials' message correctly when incorrect login details are provided.